src/recipes/heartbeat-by-user-first-impression/config.js

```diff
@@ -94,16 +150,27 @@ const engagementRules = [
   // other
 ];
 
-let supportedLocales = [ 'de',
+let supportedLocales = [
+  'cs',
+  'de',
   'en-US',
   'en-GB',
   'es',
   'es-ES',
   'es-MX',
   'fr',
+  'it',
   'zh-CN',
 ];
 
+const localeMultiplier = { //TODO: Remove this hacky, temp code
```

Comment:
It ends up being in the exported config data, under "Aurora".  I thought that this was easier to parse personally.